knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -241,27 +253,22 @@ func rawKafkaChannelManifest(apiclient client.Client) (mf.Manifest, error) {
 	return mfc.NewManifest(kafkaChannelManifestPath(), apiclient, mf.UseLogger(log.WithName("mf")))
 }
 
-func kafkaChannelManifest(instance *operatorv1alpha1.KnativeKafka, apiClient client.Client) (*mf.Manifest, error) {
-	manifest, err := rawKafkaChannelManifest(apiClient)
-	if err != nil {
-		return nil, fmt.Errorf("failed to load KafkaChannel manifest: %w", err)
-	}
-
-	manifest, err = manifest.Transform(
+func (r *ReconcileKnativeKafka) kafkaChannelManifest(instance *operatorv1alpha1.KnativeKafka, apiClient client.Client) (*mf.Manifest, error) {
```

Comment:
The `apiClient` is now unused here (and in the other function probably too)

Reply:
yep - I just checked the `kafkaSourceManifest`, IDEA reports unused

Reply:
Fixed